domains/raghavyuva.json

```diff
@@ -6,7 +6,7 @@
       },
 
       "record": {
-        "CNAME": "raghavyuva.github.io/portfolio/"
+        "CNAME": "raghavyuva.github.io"
```

Comment:
404?

Reply:
That's why It is github.com/raghavyuva/portfolio and not github.com/raghavyuva

Reply:
Please rename your portfolio repo to raghavyuva.github.io

Reply:
Changed,

Reply:
kindly check this out.